test/com/google/enterprise/gsafeed/AclTest.java

```diff
@@ -83,65 +80,53 @@ public void testGetInheritanceTypeNotSet() throws Exception {
 
   @Test
   public void testGetInheritanceTypeInvalid() throws Exception {
-    thrown.expect(IllegalArgumentException.class);
-    thrown.expectMessage("foo");
     Acl acl = unmarshal("<acl inheritance-type='foo'/>");
-    acl.getInheritanceType();
+    assertEquals(null, acl.getInheritanceType());
```

Comment:
This is the price we pay for unmarshalling to an enum? Invalid values are just silently ignored? What happens with an invalid value if we unmarshal the full XML using the DTD?

Reply:
This is what we get without validation. With validation, the parser throws an exception.